src/dashboard/src/installer/forms.py

```diff
@@ -44,5 +44,14 @@ def save(self, commit=True):
         return user
 
 
+class OrganizationForm(forms.Form):
+    """
+    Simplified version of the superuser form - simply ask for organisation info
+    """
+    org_name = forms.CharField(label=_('Organization name'), help_text=_('PREMIS agent name'), required=False, widget=TextInput(attrs=settings.INPUT_ATTRS))
+    org_identifier = forms.CharField(label=_('Organization identifier'), help_text=_('PREMIS agent identifier'), required=False, widget=TextInput(attrs=settings.INPUT_ATTRS))
```

Comment:
I'm realizing this `forms.py` should be using ugettext_lazy instead of ugettext, right?